### PR TITLE
Create init config class representation

### PR DESCRIPTION
**Pull request by katlapinka**

### CODE REVIEW

File: test/functional/api/cas/init_config.py [hunk 1]
```diff
@@ -12,40 +12,60 @@
...
+        for cache in casadm_parser.get_caches():
+            cache_lines.append(CacheConfigLine(cache.cache_id,
+                                               cache.cache_device,
+                                               cache.get_cache_mode(),
+                                               load,
+                                               extra_flags))
+            for core in casadm_parser.get_cores(cache.cache_id):
+                core_lines.append(CoreConfigLine(cache.cache_id,
+                                                 core.core_id,
+                                                 core.core_device))
```
Comment (Deixx) [inline test/functional/api/cas/init_config.py]: Consider creating (and maybe returning?) new instance of InitConfig

Comment (katlapinka) [reply test/functional/api/cas/init_config.py]: done

**Comment by micrakow** [comment]
@CAS-Linux-Jenkins retest this please

**Comment by robertbaldyga** [comment]
retest this please

**Comment by robertbaldyga** [comment]
retest this please

File: test/functional/api/cas/init_config.py [hunk 2]
```diff
@@ -12,40 +12,54 @@
...
+            for c in self.cache_config_lines:
+                config_lines.append(str(c))
+        if self.core_config_lines:
+            config_lines.append(CoreConfigLine.header)
+            for c in self.core_config_lines:
+                config_lines.append(str(c))
+        fs_utils.write_file(opencas_conf_path, '\n'.join(config_lines), False)
+
+    @staticmethod
+    def create_init_config_from_running_configuration(load: bool = None, extra_flags=""):
```
Comment (imjfckm) [inline test/functional/api/cas/init_config.py]: I think that factory method would be better as a `@classmethod`, this way inside you can just reference `cls()` and make it more robust for future refactors etc.

File: test/functional/api/cas/init_config.py [hunk 3]
```diff
@@ -12,40 +12,54 @@
...
+                                cache.get_cache_mode(),
+                                load,
+                                extra_flags)
+            for core in casadm_parser.get_cores(cache.cache_id):
+                init_conf.add_core(cache.cache_id, core.core_id, core.core_device)
+        init_conf.save_config_file()
+        return init_conf
+
+    @staticmethod
+    def create_default_init_config():
```
Comment (imjfckm) [inline test/functional/api/cas/init_config.py]: Ditto

File: test/functional/api/cas/init_config.py (hunk 2 above)
Comment (katlapinka) [reply test/functional/api/cas/init_config.py]: Done

File: test/functional/api/cas/init_config.py (hunk 3 above)
Comment (katlapinka) [reply test/functional/api/cas/init_config.py]: Done

**Pull request merged**